frontend/js/src/components/Footer.tsx

```diff
@@ -27,14 +27,13 @@ export default function Footer() {
             </p>
             <ul className="list-unstyled">
               <li className="color-a">
-                <span className="color-gray">IRC: </span>{" "}
+                <span className="color-gray">Chat with us: </span>{" "}
                 <a
-                  href="https://kiwiirc.com/nextclient/irc.libera.chat/?#metabrainz"
+                  href="https://wiki.musicbrainz.org/Communication/ChatBrainz"
```

Comment:
I think the link should go to the doc page, not directly the wiki.

https://musicbrainz.org/doc/Communication/ChatBrainz